tests/vcs/git/GitCommitBearTest.py

```diff
@@ -139,6 +139,20 @@ def test_shortlog_checks_shortlog_trailing_period(self):
         self.assertEqual(self.run_uut(shortlog_trailing_period=False), [])
         self.assertEqual(self.run_uut(shortlog_trailing_period=None), [])
 
+    def test_shortlog_checks_imperative(self):
+        self.git_commit("tag: Add shortlog in imperative")
+        self.assertNotIn("Shortlog of HEAD commit isn't imperative mood",
+                         ''.join(self.run_uut()))
```

Comment:
is there a reason why you don't use lists for comparison like the other tests above? In fact they are slightly better, as it could be that `run_uut()` spits out more than one item that yield concatenated your string (highly unlikely, but anyway). Also it's more complicated to read imo^^